src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -375,6 +375,10 @@
       <p>{{ t('You’ll have to go to a hearing. We’ll contact you with the details.') }}</p>
     {% endif %}
   </div>
+  <p class="receipt-download-container">
+    {{ internalLink(t('Download {{ claimantName }}’s hearing requirements ', { claimantName: claim.claimData.claimant.name} ), ResponsePaths.claimantDQs.evaluateUri({ externalId: claim.externalId }) )}}
+    {{ t('(PDF)') }}
```

Comment:
Please leave this line on the same line as the one above. carriage returns inside p tags are handled weirdly by some browsers. they get trimmed out rather than turned into spaces.